src/PrintInfo.jsx

```diff
@@ -17,6 +17,7 @@ export class PrintInfo extends React.Component {
   }
 }
 
+/* eslint-disable camelcase */
 export const getComplaintCountText = aggs => {
   const { doc_count, total } = aggs
```

Comment:
You can also get around this by "recasting" the name

```javascript
const { doc_count: docCount, total } = aggs
```

Reply:
thank you. I needed that.